frontend/src/components/BetaAlert.tsx

```diff
@@ -9,10 +9,7 @@ const BetaAlert = () => {
   });
 
   return (
-    <div
-      data-testid="beta-alert"
-      className="desktop:position-sticky top-0 z-200"
-    >
+    <div data-testid="beta-alert">
```

Comment:
The BetaAlert should not longer be sticky. This was done in this change, since the Header is now sticky.